app/models/manageiq/providers/redhat/network_manager/event_catcher.rb

```diff
@@ -1,4 +1,6 @@
 class ManageIQ::Providers::Redhat::NetworkManager::EventCatcher < ::MiqEventCatcher
+  include ManageIQ::Providers::Openstack::EventCatcherMixin
```

Comment:
@agrare should the method `all_valid_ems_in_zone` live in core?

Reply:
Hey @chessbyte it is: https://github.com/ManageIQ/manageiq/blob/22fbc8e0d32fde736128228783e94fb5a0e4247d/app/models/mixins/provider_worker_mixin.rb#L13-L15

Openstack adds an additional check over the standard authentication_status one which is the reason for the override.  Previously Openstack had this override copy&pasted in each of its three event catchers.  https://github.com/ManageIQ/manageiq-providers-openstack/pull/804 removes this duplication.

Reply:
I guess I find it odd that the ovirt provider is using code defined in the openstack provider.

Reply:
The Ovirt OVN network provider is an openstack neutron service, so our entire Ovirt::NetworkManager uses openstack code.